.github/workflows/test.yml

```diff
@@ -18,4 +18,4 @@ jobs:
           message: ":warning: Secrets found in repository ${{ inputs.repo-name }}"
           slack-url: ${{ secrets.SLACK_WEBHOOK }}
       - name: Run tests
-        run: sbt copySchema test
+        run: sbt copySchema copyValidationMessageProperties copyGuidanceProperties test
```

Comment:
Can we combine in one command, which will copy all the schemas and config files?